src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
 export interface Figure {
-
+  shape: 'triange' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
would be better to create enums for that - it is much easier to operate with enums than with string values